index.html

```diff
@@ -181,6 +181,27 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var fireballSize = 22;
+  var wizardSpeed = 3;
+  var wizardWidth = 70;
```

Comment:
if you have several definitions of variables with `var` keywords
use only one and the beginning
```suggestion
var  fireballSize = 22,
       wizardSpeed = 3,
       wizardWidth = 70;
```
It's the best practice

Reply:
@StireArLankar please change this and we could merge